### PR TITLE
Update deployment-prep-active-directory.md

### PR DESCRIPTION
**Pull request by steveswalwell**
Added a not around enforced GPO's still being applied when GPO inheritance is blocked, and links to using WMI Filters or security groups.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/deploy/deployment-prep-active-directory.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ Active Directory requirements for Azure Stack HCI include:
 - Group policy inheritance that is blocked for the applicable Group Policy Object (GPO).
 - A user account that has all rights to the OU in the Active Directory.
 
+> [!NOTE]
+> When group policy inheritance is blocked at the OU level this does not include any enforced GPO's.  Ensure that any applicable GPO which are enforced are also blocked using other methods, for example, using [WMI Filters](https://techcommunity.microsoft.com/t5/ask-the-directory-services-team/fun-with-wmi-filters-in-group-policy/ba-p/395648) or [security groups](https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012).
+
```
Comment (alkohli) [inline azure-stack/hci/deploy/deployment-prep-active-directory.md:26]: Change to: 

> [!NOTE]
> When group policy inheritance is blocked at the OU level, enforced GPO's aren't included.  Ensure that any applicable GPO which are enforced. are also blocked using other methods, for example, using [WMI Filters](https://techcommunity.microsoft.com/t5/ask-the-directory-services-team/fun-with-wmi-filters-in-group-policy/ba-p/395648) or [security groups](https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012).

**Review by alkohli — COMMENTED** [review]
@steveswalwell Minor changes suggested. If you could incorporate, I will sign off. Thanks again for your contribution!!

**Comment by alkohli** [comment]
@steveswalwell please see the suggested changes above. Thanks!

**Comment by alkohli** [comment]
Pulled into this PR: https://github.com/MicrosoftDocs/azure-stack-docs-pr/pull/14688. Once the PR builds, I will sign-off.

**Pull request merged**